tabpy-server/tabpy_server/psws/callbacks.py

```diff
@@ -29,78 +29,80 @@ def wait_for_endpoint_loaded(py_handler, object_uri):
-            object_type = obj_info['type']
-            object_path = get_query_object_path(
-                                settings['state_file_path'],
-                                object_name, object_version)
+            get_query_object_path(
+                settings['state_file_path'],
+                object_name, object_version)
         except Exception as e:
-            log_error('Exception encounted when downloading object: %s, error: %s' % \
-                                            (object_name, e))
+            log_error('Exception encounted when downloading object: %s'
+                      ', error: %s' % (object_name, e))
 
 
 @gen.coroutine
 def init_model_evaluator(settings):
     '''
-    This will go through all models that the service currently have and initialize them.
+    This will go through all models that the service currently have and
+    initialize them.
     '''
-    try:
-        tabpy = settings['tabpy']
-        py_handler = settings['py_handler']
+    tabpy = settings['tabpy']
+    py_handler = settings['py_handler']
 
-        existing_pos = tabpy.get_endpoints()
-        
-        for (object_name, obj_info) in (existing_pos.items() if sys.version_info > (3,0) else existing_pos.iteritems()):
-            object_version = obj_info['version']
-            object_type = obj_info['type']
-            object_path = get_query_object_path(
-                settings['state_file_path'],
-                object_name, object_version)
+    existing_pos = tabpy.get_endpoints()
 
-            log_info('Load endpoint: %s, version: %s, type: %s' % \
-                                (object_name, object_version, object_type))
-            if object_type == 'alias':
-                msg = LoadObject(object_name, obj_info['target'], object_version,
-                    False, 'alias')
-            else:
-                local_path = object_path                    
-                msg = LoadObject(object_name, local_path, object_version,
-                            False, object_type)
-            py_handler.manage_request(msg)
+    for (object_name, obj_info) in (
+            existing_pos.items() if sys.version_info > (3, 0)
+            else existing_pos.iteritems()):
+        object_version = obj_info['version']
+        object_type = obj_info['type']
+        object_path = get_query_object_path(
+            settings['state_file_path'],
+            object_name, object_version)
+
+        log_info('Load endpoint: %s, version: %s, type: %s' %
+                 (object_name, object_version, object_type))
+        if object_type == 'alias':
+            msg = LoadObject(object_name, obj_info['target'],
+                             object_version, False, 'alias')
+        else:
+            local_path = object_path
+            msg = LoadObject(object_name, local_path, object_version,
+                             False, object_type)
+        py_handler.manage_request(msg)
 
-    except Exception as e:
```

Comment:
AFAICT this try...except block had an uninitialized variable of `host_to_initialize` in the except clause, so if anything it would have just thrown another error and obfuscated what really happened